test/framework/interface.go

```diff
@@ -517,6 +518,12 @@ func WithTransparentProxyV1(transparent bool) AppDeploymentOption {
 	})
 }
 
+func WithDpEnvs(envs map[string]string) AppDeploymentOption {
```

Comment:
nit: maybe just `WithEnvs`?

Reply:
The file `interface.go` has `WithEnvs` defined already for `KumaDeploymentOption` and because both are in the same file we cannot define the one for app. Maybe we should split the file `interface` into one for app other for kuma deployment.

Reply:
oh, ok :/